card/lib/card/simplecov_helper.rb

```diff
@@ -1,3 +1,6 @@
+def is_in_mod? file,mod_path
+  file.src.join("") =~ /below pulled from #{Rails.root}\/mod/#{mod_path}\//
+end
```

Comment:
We can do this for now, but it might make more sense to put a cleaner flag in those files at the time that they're generated.